client/src/components/modals/pick-entity-or-conversion.vue

```diff
@@ -141,7 +141,7 @@ export default class PickEntityOrConversion extends Vue {
     return `${210 * cols > maxThreshold ? maxThreshold : 210 * cols}px`
   }
   entityBlurbs (entity_type_cd: string) {
-    return newReqModule.entityBlurbs.find(type => type.value === entity_type_cd)?.blurbs
+    return newReqModule.entityBlurbs && newReqModule.entityBlurbs.find(type => type.value === entity_type_cd)?.blurbs
```

Comment:
I tried to optional chain this after `newReqModule.entityBlurbs` but it didn't like that, so i went with a traditional short circuit to prevent console type errors.

Reply:
Optional chaining is a TS thing. In the template you only have JS.

Reply:
Maybe because it's a module? Dunno.

Reply:
Totally, will look into it! (when we have time 😆 )